proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+* Maintainers of the image-spec project
+* Three additional most active contributors (by commit count as of Wed, 8/24/2016) to the files that will become image-tools:
+  - Sergiusz Urbaniak (https://github.com/s-urbaniak)
+  - Antonio Murdaca (https://github.com/runcom)
+  - xiekeyang (https://github.com/xiekeyang)
+
+(Vincent Batts and Jonathan Boulle are two of the most active contributors and they will be included with the first bullet point.)
+
+### Code of Conduct
+Both of the proposed projects would incorporate (by reference) the OCI Code of Conduct
+
+### Governance and Releases
+Both of the proposed projects would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+Released version numbers of the runtime-tools and image-tools projects should roughly align with released verions of the associated specs projects.
+
+It is expected that releases of the tools projects will follow releases of the specs projects by anywhere from a few days to a few months.
+
+## Frequenty Asked Questions (FAQ)
+Q: What about the runc project?
+A: This proposal would not impact runc.  
+Runc is a reference implementation of the runtime-spec and it lives in its own repo.
+
+Q: What about schemas?
+A: It would be up to the OCI developer community to determine if schemas fit best with the specs or tools projects.
+
+Q: Why are the repo names plural (tools) instead of singular (tool)?
+A: There may be multiple tools or a single tool with multiple functionalities.
+
+Q: Why not prefix the repo names with "oci-" aligning to the CLI commands for invoking the tools?
+A: The repos will be under the http://www.github.com/OpenContainers/ organization.
+
+Q: Why two tools repos?
+A: Having a tools repo for each spec repo allows for releasing a tools repo shortly after a spec stabilizes even if the other tools repo is in a state of flux.
+
+Q: Why is contribution count used to identify potential maintainers?
+A: Contribution count is not a great metric for the value of a contributor's contributions, but this seemed like a reasonable, rough way to identify additional potential maintainers beyond the spec maintainers.  
+
+Q: Can additional code/tools be added to the proposed tools repos beyond what is listed above?
+A: Yes.  If there is other code or functionality that a contributor believes would be relevant for runtime-related or image-related tools, it would be great if it is contributed.
+
+Q: Does this change the OCI Charter or Scope Table?
+A: No.  Nothing in this proposal is intended to amend the OCI Charter or OCI Scope Table.
```

Comment:
For folks reading this proposal who may not be familiar with them already, can we link [here][1] and [here][2]?

[1]: https://www.opencontainers.org/about/governance
[2]: https://www.opencontainers.org/governance/oci-scope-table